### ISSUE TITLE
Error in `go get -u

### ISSUE THREAD

**Issue by tbpg** [census-instrumentation/opencensus-go#483]
# What I did:
```bash
$ go get -u go.opencensus.io/...
package go.opencensus.io/examples/grpc/exporter: cannot find package "go.opencensus.io/examples/grpc/exporter" in any of:
	/usr/local/go/src/go.opencensus.io/examples/grpc/exporter (from $GOROOT)
	/home/tbp/go/src/go.opencensus.io/examples/grpc/exporter (from $GOPATH)
```

This command is listed in the README.

# What I expected:
No errors.

# What I got:
package go.opencensus.io/examples/grpc/exporter: cannot find package "go.opencensus.io/examples/grpc/exporter" in any of:

**Comment by ramonza** [comment]
That's weird. I tried 

    export GOPATH=$(mktemp -d); go version && echo $GOPATH && go get -u go.opencensus.io/...
    go version go1.9 darwin/amd64

and this worked (after a good long while).

**Comment by odeke-em** [comment]
@tbpg, please get the latest code with `go get -u` like @ramonza just did in  https://github.com/census-instrumentation/opencensus-go/issues/483#issuecomment-368112295

**Comment by ramonza** [comment]
@odeke-em looks like @tbpg did use that command and it still didn't work, any ideas why that might be?

**Comment by odeke-em** [comment]
@ramonza, yeah, that code changed yesterday, from this PR https://github.com/census-instrumentation/opencensus-go/pull/476 and I also had encountered this problem and commented https://github.com/census-instrumentation/opencensus-go/pull/476#discussion_r170133982
